tests/tests.js

```diff
@@ -81,7 +71,7 @@ describe('Discord Tests', function() {
          * Test that the hook returns a 200 when posted to
          */
         it('Hook returns a 200 OK when posted to', function(done) {
-            sendPost('1', function(error, response, body) {
+            sendPost(testUtils.recordedFixturesDir + '1', function(error, response, body) {
```

Comment:
Ah, that makes sense then; just reads funny. I expected a function signature more like the `request.post` call that this wraps. (e.g., `sendPost(<url>, <body>, <callback>)`)

Could we change the function to `sendHookPost` to be more explicit that the url is hard-coded in the function?